### PR TITLE
Partials for Azure CISv1.2.0 RBAC & Security Center

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description

Update RBAC & Security Center Partials for React Frontend.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import InformationsWrapper from '../../../components/InformationsWrapper';
+import { Partial, PartialValue } from '../../../components/Partial';
+import { partialDataShape } from '../../../utils/Partials';
+
+const propTypes = {
+  data: PropTypes.shape(partialDataShape).isRequired,
+};
+
+const CustomRolesreport = props => {
```
Comment (loic294) [inline ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js:12]: typo? CustomRolesReport?

File: ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  const { data } = props;
+
+  if (!data) return null;
+
+  return (
+    <Partial data={data}>
+      <InformationsWrapper>
+        <PartialValue label="No Administering Resource Locks Role" valuePath="missing_custom_role_administering_resource_locks" />
```
Comment (loic294) [inline ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js:21]: haha what a big partial! 😂

**Review by loic294 — APPROVED** [review]
Small typo but the rest is good!

File: ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js (hunk 1 above)
Comment (xnkevinnguyen) [reply ScoutSuite/frontend/src/partials/azure/rbac.subscriptions.id.custom_roles_report.id/index.js]: fixed

**Pull request merged**